vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -1586,7 +1588,7 @@ void LedgerImpl::OnGrantViaSafetynetCheck(
 
 void LedgerImpl::ApplySafetynetToken(
     const std::string& promotion_id, const std::string& token) const {
-  bat_grants_->SetGrant("", promotion_id, token);
+//  bat_grants_->SetGrant("", promotion_id, token);
```

Comment:
@NejcZdovc Any advice on how best to handle this now that solving the captcha requires a callback?